lightkurve/correctors.py

```diff
@@ -357,6 +362,8 @@ def correct(self, time, flux, centroid_col, centroid_row,
                 flux[i] = corrected_flux
 
             flux_hat = np.append(flux_hat, flux[i])
+            if restore_trend:
+                flux_hat += self.trend
```

Comment:
Not sure if I follow the maths here. `iter_trend` is being divided out in  ``self.normflux = flux[i] / iter_trend``,
so maybe instead of ``flux_hat += self.trend`` it should be ``flux_hat *= self.trend``?.

Reply:
I think you're absolutely right!